### PR TITLE
fix: update README for library conversion to microgenerator

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,6 +32,16 @@ In order to use this library, you first need to go through the following steps:
...
 
+Supported Python Versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Python >= 3.6
+
+Deprecated Python Versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Python == 2.7.
+
+The last version of this library compatible with Python 2.7 is google-cloud-webrisk==1.0.0
```
Comment (busunkim96) [inline README.rst:43]:
```suggestion
The last version of this library compatible with Python 2.7 is google-cloud-webrisk==0.3.0 
```

Comment (xiaozhenliu-gg5) [reply README.rst:43]: curious that in setup.py, the release version is 1.0.0, which one should we take here? Thanks

Comment (busunkim96) [reply README.rst]: @xiaozhenliu-gg5 The microgenerator code has been merged and released (as 1.0.0). #25 The previous version was 0.3.0

Comment (xiaozhenliu-gg5) [reply README.rst]: 👍 sounds reasonable.

**Pull request merged**